### PR TITLE
feat: updates and enhancements

### LINKED ISSUE #4014: sigma/rules/windows/process_creation/proc_creation_win_susp_outlook.yml - Exclusion required for Gpg4win
Hi,

As described here (https://www.gpg4win.org/about.html):
> Gpg4win enables users to securely transport emails and files with the help of encryption and digital signatures. Encryption protects the contents against an unwanted party reading it. Digital signatures make sure that it was not modified and comes from a specific sender.
Gpg4win supports both relevant cryptography standards, OpenPGP and S/MIME (X.509), and is the official GnuPG distribution for Windows. It is maintained by the developers of GnuPG. Gpg4win and the software included with Gpg4win are [Free Software](http://fsfe.org/about/basics/freesoftware.en.html) (Open Source; among other things free of charge for all commercial and non-commercial purposes).

I installed Gpg4win 4.1.0 on Microsoft Office Outlook 2301.

Could you please provide an exclusion to avoid alerts on activity listed below?

| Rule_Title 	| Suspicious Execution from Outlook 	|
|:---:	|---	|
| Rule_Id 	| e212d415-0e93-435f-9e1a-f29005bb4723 	|
| Rule_Description 	| Detects EnableUnsafeClientMailRules used for Script Execution from Outlook 	|
| Rule_Author 	| Markus Neis 	|
| Rule_Modified 	| 2022/01/07 	|
| Rule_References 	| https://github.com/sensepost/ruler https://www.fireeye.com/blog/threat-research/2018/12/overruled-containing-a-potentially-destructive-adversary.html 	|
| Rule_Level 	| high 	|
| Rule_FalsePositives 	| Unknown 	|
| Rule_Link 	| https://github.com/SigmaHQ/sigma/blob/0.22-1988-gfa21f5ae5/rules/windows/process_creation/proc_creation_win_susp_outlook.yml 	|
| Rule_Path 	| public\windows\process_creation\proc_creation_win_susp_outlook.yml 	|
| Rule_Sigtype 	| public 	|

Matched Strings Info:
| ParentImage 	| C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE 	|
|:---:	|---	|
| CommandLine1 	| "C:\\\\\\\\\\\\\\\\\\\\\\\\Program Files (x86)\Gpg4win\bin\gpgme-w32spawn.exe.exe" "C:\\\\\\\\\\\\\\\\\\\\Users\\username\\AppData\\Local\\Temp\\gpgme-58dKij" "C:\\Program Files (x86)\\GnuPG\\bin\\gpgconf.exe" "--list-components"  	|
| CommandLine2 	| "C:\\\\\\\\\\\\\\\\\\\\\\\\Program Files (x86)\Gpg4win\bin\gpgme-w32spawn.exe.exe" "C:\\\\\\\\\\\\\\\\\\\\Users\\username\\AppData\\Local\\Temp\\gpgme-sK8W5s" "C:\\Program Files (x86)\\GnuPG\\bin\\gpgconf.exe" "--list-dirs"  	|
| CommandLine3 	| "C:\\\\\\\\\\\\\\\\\\\\\\\\Program Files (x86)\Gpg4win\bin\gpgme-w32spawn.exe.exe" "C:\\\\\\\\\\\\\\\\\\\\Users\\username\\AppData\\Local\\Temp\\gpgme-pzzN7G" "C:\\Program Files (x86)\\GnuPG\\bin\\gpgconf.exe" "--version" 	|

Thanks!

Regards,
WikiJM

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Resolves #4014
- Update rules related to "onenote" exploitation
- Various other updates and quality of life improvements

### CODE REVIEW

File: rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml [hunk 1]
```diff
@@ -1,13 +1,16 @@
 title: Unusual File Download from File Sharing Domain
```
Comment (frack113) [inline rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml:1]:
```suggestion
title: Unusual File Download from WEB File Sharing Domain
```

File: rules/windows/process_creation/proc_creation_win_office_outlook_susp_child_processes_remote.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+title: Suspicious Outlook Remote Child Process
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_office_outlook_susp_child_processes_remote.yml:1]:
```suggestion
title: Suspicious Remote Child Process from Outlook 
```
At first reading I thought it was outlook the remote process.

File: rules/windows/process_creation/proc_creation_win_office_outlook_susp_child_processes_remote.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+id: e212d415-0e93-435f-9e1a-f29005bb4723
+related:
+    - id: 208748f7-881d-47ac-a29c-07ea84bf691d # Outlook Child Processes
+      type: similar
+status: test
+description: Detects a suspicious child process spawning from outlook where the image is located in a remote location (SMB/WebDav shares).
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_office_outlook_susp_child_processes_remote.yml:7]:
```suggestion
description: Detects a suspicious child process spawning from Outlook where the image is located in a remote location (SMB/WebDav shares).
```

File: rules/windows/process_creation/proc_creation_win_susp_outlook.yml [hunk 4]
```diff
@@ -1,30 +0,0 @@
-title: Suspicious Execution from Outlook
-id: e212d415-0e93-435f-9e1a-f29005bb4723
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_outlook.yml:2]: obligatory `git mv` reminder

**Review by phantinuss — APPROVED** [review]

File: rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml (hunk 1 above)
Comment (nasbench) [reply rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml:1]: It's not a "web file sharing" it's a "file sharing" so can't use web here.

Comment (nasbench) [reply rules/windows/create_stream_hash/create_stream_hash_file_sharing_domains_download_unusual_extension.yml:1]: I wil replace `Domain` with `Website` as discussed

**Review by frack113 — APPROVED** [review]

**Pull request merged**